pkg/util/taints/taints_test.go

```diff
@@ -562,10 +737,50 @@ func TestParseTaints(t *testing.T) {
 			expectedErr: true,
 		},
 		{
-			name:        "invalid spec format for adding taint",
+			name:        "invalid spec taint value",
+			spec:        []string{"foo=nospecialchars%^@:NoSchedule"},
```

Comment:
It'd be helpful to put the "no special chars" as part of the test name itself.
Same for the below cases as well